sdk/src/main/scala/coop/rchain/sdk/casper/Stake.scala

```diff
@@ -0,0 +1,14 @@
+package coop.rchain.sdk.casper
```

Comment:
I'm not sure where is the best place to locate this functions. My motivation was to place it to *sdk* project as common project and in *casper* subfolder as stake is part of Casper algorithm.

Reply:
I think it can be in package object in casper folder. This is general logic which is used in many places. 

Also it makes sense to add extensions/aliases for bonds map, when instead of full stake it is Map[Validator, Long]. This one can be a bit tricky because you have to account situation with equal stake. If you use conversion to Set you might lose some stake, I encountered this once. So just want to warn.

Reply:
This package should be called `consensus` because it's not Casper specific. Supermajority of stake is theoretical requirement for consensus safety.